megamek/src/megamek/codeUtilities/MathUtility.java

```diff
@@ -96,12 +102,10 @@ public static int clamp(final int value, final int min, final int max) {
      * @param value the double value to clamp
      * @param min   the minimum limit
      * @param max   the maximum limit
-     * @return The value if it is inside the range given by the limits (inclusive);
-     *         the min value if value is below that range and the max value if value
-     *         is above that range.
-     *         clamp(2, 6, 8) returns 6,
-     *         clamp(7, 6, 8) returns 7,
-     *         clamp(12, 3, 5) returns 5.
+     *
+     * @return The value if it is inside the range given by the limits (inclusive); the min value if value is below that
+     *       range and the max value if value is above that range. clamp(2, 6, 8) returns 6, clamp(7, 6, 8) returns 7,
+     *       clamp(12, 3, 5) returns 5.
```

Comment:
Same comment for subsequent entries.

Reply:
Easy enough to revert back and put into a list form.